client/flutter/lib/api/content/content_store.dart

```diff
@@ -76,6 +76,15 @@ abstract class _ContentStore with Store implements Updateable {
   @observable
   PosterContent symptomPoster;
 
+  Future<void> updateBundle<T extends ContentBase>(String name, T old,
+      T Function(ContentBundle) constructor, void Function(T) setter) async {
+    final newValue =
+        constructor(await service.load(locale, countryIsoCode, name));
+    if (newValue?.bundle?.contentVersion != old?.bundle?.contentVersion) {
```

Comment:
question: do we want only newer content versions, or any different content version?
```suggestion
    if (newValue?.bundle?.contentVersion > old?.bundle?.contentVersion) {
```

Reply:
Sure we can do that, I was thinking that it is enforced in the cache manager, but this is fine too.